gcc/config/riscv/riscv.cc

```diff
@@ -9427,6 +9427,23 @@ riscv_sched_adjust_cost (rtx_insn *insn, int dep_type, rtx_insn *dep_insn, int c
   return new_cost;
 }
 
+static int
+riscv_sched_adjust_priority (rtx_insn *insn, int priority)
+{
+  if (!riscv_is_micro_arch (arcv_rhx100))
+    return priority;
+
```

Comment:
you may write an explanation why 2 such couples need higher priority. perhaps has to do with the chance to fuse them.

Reply:
Added a comment.